src/azure/com/microsoftopentechnologies/intellij/ui/libraries/LibrariesConfigurationDialog.java

```diff
@@ -103,6 +105,8 @@ private void addLibrary() {
                 newLibraryModel.commit();
                 modifiableModel.commit();
                 ((DefaultListModel) librariesList.getModel()).addElement(azureLibrary);
+            } catch (Exception ex) {
+                ex.printStackTrace();
```

Comment:
Is it ok to ignore the exception here? Also, we're catching the generic `Exception` type, can it be a more specific one?